content/scalability/docs/cluster-services.md

```diff
@@ -0,0 +1,60 @@
+
+You can reduce the number of requests to CoreDNS by lowering the ndots option (https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-dns-config) of your workload or fully qualifying your domain requests by including a trailing . (e.g. api.example.com.). If your workload connects to external services via DNS we recommend setting ndots to 2 so workloads do not make unnecessary, cluster DNS queries inside the cluster. You can set a different DNS server and search domain if the workload doesn’t require access to services inside the cluster.
+
+```
+spec:
+  dnsPolicy: "None"
+  dnsConfig:
+    options:
+      - name: ndots
+        value: "2"
+      - name: edns0
+```
+
+If you lower ndots to a value that is too low or the domains you are connecting to do not include enough specificity (including trailing .) then it is possible DNS lookups will fail. Make sure you test how this setting will impact your workloads.
+
+### Scale CoreDNS Horizontally 
+
+CoreDNS instances can scale using a horizontal pod autoscaler rule or you can use [NodeLocal DNS](https://kubernetes.io/docs/tasks/administer-cluster/nodelocaldns/). Scaling replicas horizontally is a good option to avoid failed DNS requests. NodeLocal DNS will require more compute resources in the cluster, but it will avoid failed DNS requests and increase the response time for DNS queries in the cluster.
+
+You can also use the [horizontal proportional scaler](https://github.com/kubernetes-sigs/cluster-proportional-autoscaler) to scale CoreDNS based on the number of nodes or cores in the cluster. This isn’t a direct correlation to request queries, but can be useful depending on your workloads. 
+
+## Scale Kubernetes Metrics Server Vertically
+
+The Kubernetes Metrics Server supports horizontal and vertical scaling. By horizontally scaling the Metrics Server it will be highly available, but it will not scale horizontally to handle more cluster metrics. You will need to vertically scale the Metrics Server based on [their recommendations](https://kubernetes-sigs.github.io/metrics-server/#scaling) as nodes and collected metrics are added to the cluster.
+
+The Metrics Server keeps the data it collects, aggregates, and serves in memory. As a cluster grows, the amount of data the Metrics Server stores increases. In large clusters the Metrics Server will require more compute resources than the memory and CPU reservation specified in the default installation. You can use [Addon Resizer](https://github.com/kubernetes/autoscaler/tree/master/addon-resizer) to scale the Metrics Server vertically in proportion to the size of your worker nodes. 
```

Comment:
I think we need to offer an explanation about why we're not recommend VPA here. This is a legitimate recommendation, but it would be nice to know why we should use the addon resizer specifically for CoreDNS.